mlmodel/format/NeuralNetwork.proto

```diff
@@ -23,25 +22,31 @@
  * and a layer may not have input or output names that
  * refer to layers that are not yet defined.
  *
- *
  * For CoreML specification version <=3,
  * all inputs are mapped to static rank 5 tensors, with axis notations
  * [Sequence, Batch, Channel, Height, Width].
  *
  * From specification version 4 onwards (iOS >= 13, macOS >= 10.15), more options are available
- * (see enums "NeuralNetworkMultiArrayShapeMapping", "NeuralNetworkImageShapeMapping")
+ * (see enums ``NeuralNetworkMultiArrayShapeMapping``, ``NeuralNetworkImageShapeMapping``)
  * to map inputs to generic N-Dimensional (or N rank) tensors, where N >= 1.
  *
  * Each layer type may have specific constraints on the ranks of its inputs and outputs.
  *
+ * You can often see layers in 3 different variations: ``layernameLayerLike``, ``layernameLayerStatic``,
```

Comment:
I would change the wording slightly to (so that we can omit "you")

Several operations are available as layers in 3 different variations: ``like``, ``static`` and ``dynamic``. 
The `static` layers have a property corresponding to the shape _(is it always the output shape?_)
The ``like`` layers take an additional tensor as input, which is only used to infer the shape. Whereas in the `dynamic` case,  and additional input corresponding explicitly to the shape is expected.